schedule/migration_zdup_offline_sle12sp2_staging.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name:           migration_zdup_offline_sle12sp2_staging
+description:    >
+  Test for zdup on sle12sp2 at staging.
+vars:
+  HDDVERSION: 12-SP2
+  ZDUP: 1
```

Comment:
I do not understand why these vars are specified here. They already set in the Test Suite on openQA.

@rwx788, should the 'vars' section be used to duplicate the values from Test Suites? Or it is planned to replace Test Suite variables by the vars from this section of yaml scheduling?

Reply:
It is intended to cleanup as much as possible variables in openQA webUI and move them to yaml, but not all of them are possible to be moved as stated in the doc.